changelogs/unreleased/bump-libcurl-to-8.5.0.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/build
+
+* Updated libcurl to version 8.5.0.
```

Comment:
Before release it's better to change existing note https://github.com/tarantool/tarantool/blob/master/changelogs/unreleased/bump-libcurl-to-8.4.0.md

To avoid:
* Updated libcurl to version 8.3.0.
* Updated libcurl to version 8.4.0.
* Updated libcurl to version 8.5.0.

In scope of single release

Reply:
Seems it will a first bump of libcurl in 3.1, so this comment is irrelevant now. I got you and will leave only the latest bump entry in changelog in a future.